utils/cleaners.py

```diff
@@ -358,6 +356,39 @@ def cuckoo_clean_lower_score(args):
     resolver_pool.map(lambda tid: delete_data(tid), id_arr)
 
 
+def tmp_clean_before_day(args):
+    """Clean up tmp folder
+    It deletes all items in tmp folder before now - days.
+    """
+    if not args.delete_tmp_items_older_than_days:
+        log.info("No days argument provided bailing")
+        return
+
+    days = args.delete_tmp_items_older_than_days
+    init_console_logging()
+
+    today = datetime.today()
+    tmp_folder_path = cuckoo.cuckoo.get("tmppath")
+
+    for root, directories, files in os.walk(tmp_folder_path, topdown=True):
+        for name in files + directories:
+            path = os.path.join(root, name)
+            last_modified_time_in_seconds = os.stat(os.path.join(root, path)).st_mtime
+            file_time = today - datetime.fromtimestamp(last_modified_time_in_seconds)
+
+            if file_time.days > days:
+                try:
+                    if os.path.isdir(path):
+                        log.info("Delete file: {0}".format(path))
```

Comment:
I think the info message is switched between the two conditions, this is the folder condition